kotlin/legacy/legacy-workflow-core/src/test/java/com/squareup/workflow/legacy/ReactorAsWorkflowIntegrationTest.kt

```diff
@@ -372,6 +375,7 @@ class ReactorAsWorkflowIntegrationTest {
     assertFalse(resultSub.isCompleted)
 
     workflow.sendEvent("foo")
+    assertEquals(SecondState("foo"), stateSub.poll())
```

Comment:
Is this line for added coverage, or did the `poll()` call actually fix something?

Reply:
Conflated channels now hang onto their last value even after they're closed, so we have to pull it out first now.

See https://github.com/Kotlin/kotlinx.coroutines/issues/332